### ISSUE TITLE
Classification over selected features with pipeline

### ISSUE THREAD

**Issue by henrique-voni** [rasbt/mlxtend#734]
Hello,

I am running some experiments with `SequentialFeatureSelector`. I am concerned about bias/overestimate my model, so i did the following: 

- Split  train/test datasets
- Scale data in training
- Run `SequentialFeatureSelector` over train dataset (cv=5)
- Train a new model with best features
- Scale test dataset
- Predict test dataset

I'd like to know how to set SequentialFeatureSelector inside the Pipeline in this case. I tried to follow the example in the documentation, but i don't understand what the selector does inside the pipeline. Other examples/explanations would be very welcome.

Thanks in advance!

Ps: congratulations on building such a great tool 😄

**Comment by rasbt** [comment]
Glad to hear that the SFS is useful. So when I understand correctly, you'd just like to embed it in a pipeline as intermediate step? Not sure if you meant this example, but Example 8 would be an example of that: http://rasbt.github.io/mlxtend/user_guide/feature_selection/SequentialFeatureSelector/#example-8-sequential-feature-selection-and-gridsearch

In essence, if you call `fit` on the scikit-learn pipeline, it would perform the feature selection and return the subset of the selected features to the next step in the pipeline (i.e., execute `fit` and then `transform` on the SFS). If you call `predict` on the pipeline, the SFS will just do the transform step. 

In short, the SFS will behave like other dim reduction techniques in scikit-learn, like PCA.

I hope this helps clarifying
